solutions/gold/cses-1745.mdx

```diff
@@ -118,9 +114,34 @@ public class MoneySums {
 		for (int i : possible) { io.print(i + " "); }
 		io.close();
 	}
+
+	// CodeSnip{Kattio}
 }
 ```
 
 </JavaSection>
+<PySection>
+
+```py
+n = int(input())
+money = list(map(int, input().split()))
+
+max_sum = n * 1000
+dp = [[False] * (max_sum + 1) for _ in range(n + 1)]
+dp[0][0] = True
```

Comment:
show proof of ac for this _updated_ solution

Reply:
![Uploading Screenshot from 2024-05-21 22-38-34.png…]()

Reply:
![Screenshot from 2024-05-21 22-38-34](https://github.com/cpinitiative/usaco-guide/assets/79394137/4d9343a6-f790-4679-ba4a-6742794ad85b)